pom.xml

```diff
@@ -1457,6 +1458,17 @@
       </dependency>
 
       <dependency>
+        <groupId>org.gwtbootstrap3</groupId>
+        <artifactId>gwtbootstrap3</artifactId>
+        <version>${version.org.gwtbootstrap3}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.gwtbootstrap3</groupId>
+        <artifactId>gwtbootstrap3-extras</artifactId>
+        <version>${version.org.gwtbootstrap3}</version>
+      </dependency>
```

Comment:
Please see my comment for UF PR (https://github.com/uberfire/uberfire/pull/444#issuecomment-233360918), this should not be necessary anymore